arbitrator/bench/src/bin.rs

```diff
@@ -18,13 +18,13 @@ use prover::prepare::prepare_machine;
 #[derive(Parser, Debug)]
 #[command(author, version, about, long_about = None)]
 struct Args {
-    /// Path to a preimages text file
+    /// Path to a preimages json file
     #[arg(short, long)]
-    preimages_path: PathBuf,
+    json_inputs: PathBuf,
 
     /// Path to a machine.wavm.br
     #[arg(short, long)]
-    machine_path: PathBuf,
+    binary: PathBuf,
```

Comment:
## OPTIONAL

This might be an unpopular opinion, but I actually prefer the name `machine_bin` to `binary` as it is more specific. Would you consider changing the name in the prover's command-line flags instead? I suppose that might also involve changing some other places that have hard-coded the flag name.